test_unstructured/metrics/test_element_type.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from unstructured.metrics.element_type import get_element_type_frequency
+from unstructured.partition.auto import partition
+
+
+@pytest.mark.parametrize(
+    ("filename", "frequency"),
+    [
+        (
+            "fake-email.txt",
+            {
+                "UncategorizedText": {"None": 6},
+                "ListItem": {"None": 12},
+                "Title": {"None": 5},
+                "NarrativeText": {"None": 2},
```

Comment:
Oh hmm, if the result is still going to be a nested dictionary whether or not the category has a depth... it's looking a little clunky. I'm leaning baking both the category name and depth into a tuple now for easier indexing (with a tuple, you can just index for the first element and ignore the second field, but with a nested dictionary you are forced to index each level). Will edit my previous comment lol